pkgs/servers/sql/sqld/default.nix

```diff
@@ -1,35 +1,45 @@
-{ lib
-, rustPlatform
-, fetchFromGitHub
-, pkg-config
-, protobuf
-, openssl
-, sqlite
-, zstd
-, stdenv
-, darwin
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  protobuf,
+  openssl,
+  sqlite,
+  zstd,
+  stdenv,
+  darwin,
+  cmake,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "sqld";
-  version = "0.17.2";
+  version = "0.24.18";
 
   src = fetchFromGitHub {
-    owner = "libsql";
-    repo = "sqld";
-    rev = "v${version}";
-    hash = "sha256-KoEscrzkFJnxxJKL/2r4cY0oLpKdQMjFR3daryzrVKQ=";
+    owner = "tursodatabase";
+    repo = "libsql";
+    rev = "libsql-server-v${version}";
+    hash = "sha256-/0Sk55GBjUk/FeIoq/hGVaNGB0EM8CxygAXZ+ufvWKg=";
   };
 
+  cargoBuildFlags = [
+    "--bin"
+    "sqld"
+  ];
+
   cargoLock = {
-    lockFile = ./Cargo.lock;
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This is import-from-derivation, which is not allowed in Nixpkgs.

Reply:
What's the best way to do this then?

Reply:
I copied it from another package 😅

Reply:
Which other package?  I don't see any other occurrence of `lockFile = "${src}` in Nixpkgs, and it shouldn't get past CI.

The right way to do it is the way it was done before, by including Cargo.lock in Nixpkgs.

Reply:
Oh, it was a search for `lang:nix lockfile`, so it wasn't from nixpkgs; my mistake.

Rebasing and fixing now.